ci/travis.sh

```diff
@@ -27,10 +27,19 @@ echo "--------------------------------------------------------------------------
 echo "-- Dialog, intents from XLS to XML, CSV";
 echo "--------------------------------------------------------------------------------";
 mkdir -p tests/data/dialog/generated;
-python scripts/dialog_xls2xml.py -x tests/data/xls/E_CZ_T2C_authoring.xlsx -gd tests/data/dialog/generated -gi "tests/data/intents" -ge "tests/data/entities" -v;
+python scripts/dialog_xls2xml.py -x tests/data/xls -gd "tests/data/dialog/generated" -gi "tests/data/intents" -ge "tests/data/entities" -v;
```

Comment:
the -x argument means the folder to scan. The script just printed error, that the xlsx file is not a directory or that something like that and quit.

Reply:
-x accepts files as well. Quotation marks added.